TestCntlrApp/build/fw.mak

```diff
@@ -69,7 +69,12 @@ BIN_DIR=../bin
 LIB_DIR=../lib
 
 # pcap archive name
-PCAP_ARC=libpcap-64bit.a
+UNAME_M := $(shell uname -m)
```

Comment:
We are evaluating this UNAME_M twice in this file. We can evaluate once and use at both the places for comparing it

Reply:
Done.

Reply:
LGTM. Just need to confirm if sanity is passing over this PR